lib/reporting/identity_verification_report.rb

```diff
@@ -206,6 +213,8 @@ def data
             event_users[Results::IDV_REJECT_VERIFY] << user_id if success == '0'
           when Events::IDV_PHONE_FINDER_RESULTS
             event_users[Results::IDV_REJECT_PHONE_FINDER] << user_id if success == '0'
+          when Events::FRAUD_REVIEW_PASSED
+            event_users[Results::FRAUD_REVIEW_PASSED] << user_id if success == '1'
```

Comment:
we could also add this filtering to the CW query itself? similar pattern as these:

```
        | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1)
                 or (name != %{usps_enrollment_status_updated})
```

in hindsight the other branches here could have maybe been done similarly

Reply:
I also just realized. this branch is redundant because we have

```
          event_users[event] << user_id
```

above. and since these are sets, it will get added anyways